### ISSUE TITLE
Grove 超音波測距感應器積木 編譯失敗

### ISSUE THREAD

**Issue by pablosun** [MediaTek-Labs/BlocklyDuino-for-LinkIt#199]
積木：
![image](https://user-images.githubusercontent.com/12759919/42549804-b2508f6c-8500-11e8-8e34-66dcf9d07617.png)

編譯錯誤訊息：
```
test_grove_us_mac:11: error: no matching function for call to 'Ultrasonic::Ultrasonic(int)'
 Ultrasonic __ultrasonic2(2);
                           ^
```

**Comment by pablosun** [comment]
The root cause is this issue:
https://github.com/MediaTek-Labs/linkit-7697-peripheral-drivers-for-arduino/issues/3

**Comment by pablosun** [comment]
Implemented in v3.0.214b
